Userland/Libraries/LibGL/GL/gl.h

```diff
@@ -255,6 +257,10 @@ extern "C" {
 // OpenGL State & GLGet
 #define GL_MODELVIEW_MATRIX 0x0BA6
 
+// Boolean macros
+#define GL_TRUE 1
+#define GL_FALSE 0
```

Comment:
Um, we already have `GL_TRUE` and `GL_FALSE`

Reply:
Uhhh, whoops